accepted/future-releases/set-literals/implementation-plan.md

```diff
@@ -0,0 +1,102 @@
+# Implementation Plan for Set literals
+
+Relevant documents:
+ - [Tracking issue](TODO)
+ - [Full proposal](https://github.com/dart-lang/language/blob/master/accepted/future-releases/set-literals/feature-specification.md)
+
+## Implementation and Release plan
+
+### Release flags
+
+The implementation of this change will happen behind
+an
+[*experiments flag*](https://github.com/dart-lang/sdk/blob/master/docs/process/experimental-flags.md).
+Tools need to be passed the flag `--enable-experiment=set-literals` for the
```

Comment:
Maybe keep the flag singular (`set-literal`)?

Reply:
OK by me, done.